osf/models/node.py

```diff
@@ -323,10 +320,22 @@ class AbstractNode(DirtyFieldsMixin, TypedModel, AddonModelMixin, IdentifierMixi
                                     related_name='parent_nodes')
 
     files = GenericRelation('osf.OsfStorageFile', object_id_field='target_object_id', content_type_field='target_content_type')
+    groups = {
```

Comment:
GuardianMixin expects groups to be defined. Here's where you tell the mixin which permissions to give to which group.